src/services/dumpers/templates/agent-nodejs/typescript/index.hbs

```diff
@@ -0,0 +1,94 @@
+import type { Schema } from './typings';
```

Comment:
Why is this file so different from the previous one?
Shouldn't it be almost the exact same ?

Reply:
I saw with @romain-gilliotte, and we feel this one is far better. We plan to update the javascript one right after this epic is closed.